src/types/mod.rs

```diff
@@ -2238,6 +2238,32 @@ pub struct NodeInfo {
     pub details:      NodeDetails,
 }
 
+/// Information of a baker for a certain reward period.
+#[derive(Debug)]
+pub struct BakerRewardPeriodInfo {
+    /// Metadata for the baker.
+    pub baker:             BakerInfo,
+    /// The stake of the baker that the
+    /// consensus protocol uses to determine lottery weight.
+    /// I.e. this is the stake after applying leverage bound and caps.
```

Comment:
Don't start sentences with `i.e.`. If you just omit it here the meaning stays the same.